supply/lib/supply/uploader.rb

```diff
@@ -106,14 +106,14 @@ def perform_upload_meta(version_codes, track_name)
       end
     end
 
-    def fetch_track_and_release!(track, version_code, status = nil)
+    def fetch_track_and_release!(track, version_code, only_statuses = nil)
```

Comment:
nitpick: maybe naming it `statuses` would be enough 🤔

Reply:
Addressed in https://github.com/fastlane/fastlane/pull/28960/commits/dc60344fc98ccd04688fd275223da2a484f6f275